description/src/lib.rs

```diff
@@ -46,61 +82,80 @@ mod tests {
             }"}
         );
     }
-    // todo!("This test with the generics does not fly yet.")
```

Comment:
Is this a regression introduced by this feature? I would enable it; and then fix it later rather than disabling, what do you think?